### PR TITLE
Add pronouns in profile

### LINKED ISSUE #238: Add pronouns in profile
# Feature Description

Add pronouns in profile (optional)

# Expected solution

Add pronouns in profile as optionally required.

# Testing

### PR DESCRIPTION
**Pull request by rakshitriya**
### Related Issue
Adds pronouns during sign up and profile page as an optional field.
Closes #238

### Testing
image

### Stakeholders
@dherbst , could you please check this?

### CODE REVIEW

File: web/admin/src/components/VolunteerRegistrationForm.vue [hunk 1]
```diff
@@ -45,6 +45,12 @@
                 </div>
               </div>
             </div>
+            <div class="col-md-6">
+              <div class="mb-3">
+                <label class="form-label">Pronoun</label>
```
Comment (dherbst) [inline web/admin/src/components/VolunteerRegistrationForm.vue:50]:
```suggestion
                <label class="form-label">Pronouns</label>
```

File: web/admin/src/views/ProfilePage.vue [hunk 2]
```diff
@@ -134,6 +135,12 @@ onBeforeMount( async () => {
         <input type="text" id="lastname" class="form-control" v-model="profile.lastname">
       </div>
     </div>
+    <div class="row my-3">
+      <div class="col">
+        <label class="form-label" for="pronoun">Pronoun</label>
```
Comment (dherbst) [inline web/admin/src/views/ProfilePage.vue:140]:
```suggestion
        <label class="form-label" for="pronoun">Pronouns</label>
```

File: web/admin/src/views/ProfilePage.vue [hunk 3]
```diff
@@ -134,6 +135,12 @@ onBeforeMount( async () => {
...
+        <input class="form-control" v-model="profile.pronoun" autocomplete="pronoun">
```
Comment (dherbst) [inline web/admin/src/views/ProfilePage.vue:141]:
```suggestion
        <input class="form-control" v-model="profile.pronoun" autocomplete="pronouns" placeholder="(she/her, they/them, he/him, etc)">
```

**Review by dherbst — CHANGES_REQUESTED** [review]
Consider changing the word "pronoun" to "pronouns" because there is more than one expected.

I also added a placeholder as a hint to the user, similar to what slack does: (she/her, they/them, he/him, etc)

**Comment by rakshitriya** [comment]
Committed your changes!

File: web/admin/src/components/VolunteerRegistrationForm.vue [hunk 4]
```diff
@@ -45,6 +45,12 @@
                 </div>
               </div>
             </div>
+            <div class="col-md-6">
+              <div class="mb-3">
+                <label class="form-label">Pronouns</label>
+                <input class="form-control" v-model="profile.pronoun" autocomplete="phone">
```
Comment (dherbst) [inline web/admin/src/components/VolunteerRegistrationForm.vue:51]:
```suggestion
                <input class="form-control" v-model="profile.pronoun" placeholder="(she/her, they/them, he/him, etc)">
```
There isn't an autocomplete for pronouns. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

File: web/admin/src/views/ProfilePage.vue [hunk 5]
```diff
@@ -134,6 +135,12 @@ onBeforeMount( async () => {
         <input type="text" id="lastname" class="form-control" v-model="profile.lastname">
       </div>
     </div>
+    <div class="row my-3">
+      <div class="col">
+        <label class="form-label" for="pronoun">Pronouns</label>
+        <input class="form-control" v-model="profile.pronoun" autocomplete="pronouns" placeholder="(she/her, they/them, he/him, etc)">
```
Comment (dherbst) [inline web/admin/src/views/ProfilePage.vue:141]:
```suggestion
        <input class="form-control" v-model="profile.pronoun" placeholder="(she/her, they/them, he/him, etc)">
```

**Review by dherbst — APPROVED** [review]
lgtm

**Comment by dherbst** [comment]
Thanks for doing this one!

**Comment by rakshitriya** [comment]
Thankyou for giving me a chance! 🎉

**Pull request merged**